python/lsst/qa/explorer/writeObjectTable.py

```diff
@@ -84,7 +84,7 @@ def readCatalog(self, patchRef):
             catalogDict[dataset] = catalog
         return filterName, catalogDict
 
-    def mergeCatalogs(self, catalogs, patchRef):
+    def run(self, catalogs, patchRef):
```

Comment:
This method takes a data reference (called `patchRef`; even though it's not documented below, it's in the `def`). So shouldn't this be called `runDataRef`, and a separate `run` method created?

Reply:
The parent class `MergeSourcesTask` has a `runDataRef` method which then calls `run` and subclasses must provide a definition for `run`.  You can argue about whether the parent class's old `mergeCatalogs` method should have been renamed to `run` in `pipe_tasks`, but this method can't be named `runDataRef`.